Gemfile

```diff
@@ -45,3 +45,4 @@ else
   gem 'sorbet'
   gem 'sorbet-runtime'
 end
+gem 'sorbet-coerce'
```

Comment:
I think this should be listed in gemspec

Reply:
It's in the gemspec already. Do we still need to list it in Gemfile?